File: GooglePay/src/main/java/com/braintreepayments/api/GooglePayTokenizeCallback.java
package com.braintreepayments.api;


import androidx.annotation.Nullable;

/**
 * Callback for receiving result of
 * {@link GooglePayClient#tokenize(GooglePayPaymentAuthResult, GooglePayTokenizeCallback)}.
 */
public interface GooglePayTokenizeCallback {

    /**
     * @param googlePayResult {@link PaymentMethodNonce}
     */
    void onResult(@Nullable GooglePayResult googlePayResult);
}

File: GooglePay/src/main/java/com/braintreepayments/api/GooglePayTokenizeCallback.kt
package com.braintreepayments.api

/**
 * Callback for receiving result of [GooglePayClient.tokenize].
 */
fun interface GooglePayTokenizeCallback {

    /**
     * @param googlePayResult [PaymentMethodNonce]
     */
    fun onGooglePayResult(googlePayResult: GooglePayResult?)
}
